packages/flame/lib/src/geometry/ray2.dart

```diff
@@ -121,4 +121,7 @@ class Ray2 {
     this.origin.setFrom(origin);
     this.direction = direction;
   }
+
+  @override
+  String toString() => 'origin: $origin, direction: $direction';
```

Comment:
This should include `Ray2`.

Reply:
Will fix, still debugging some issues with the ray tracing on this one.